.gitignore

```diff
@@ -13,4 +13,5 @@ config.yaml
 *.bak
 openshift-install*
 data/pull-secret
-dist
+bin/
+dist
```

Comment:
New empty line?

Reply:
yeah... it's github "feature" :-/ ...  when resolving conflict on the web, it will not add the new line as normal editor did... will fix it.